apps/api/src/app/events/e2e/bridge-trigger.e2e.ts

```diff
@@ -72,111 +58,107 @@ contexts.forEach((context: Context) => {
       await bridgeServer.stop();
     });
 
-    it(`should trigger the bridge workflow with sync [${context.name}]`, async () => {
-      const workflowId = `hello-world-${`${context.name}`}`;
-      const newWorkflow = workflow(
-        workflowId,
-        async ({ step, payload }) => {
-          await step.email(
-            'send-email',
-            async (controls) => {
-              return {
-                subject: `This is an email subject ${controls.name}`,
-                body: `Body result ${payload.name}`,
-              };
-            },
-            {
-              controlSchema: {
-                type: 'object',
-                properties: {
-                  name: { type: 'string', default: 'TEST' },
-                },
-              } as const,
-            }
-          );
-
-          await step.inApp(
-            'send-in-app',
-            async (controls) => {
-              return {
-                body: `in-app result ${payload.name}`,
-              };
-            },
-            {
-              controlSchema: {
-                type: 'object',
-                properties: {
-                  name: { type: 'string', default: 'TEST' },
-                },
-              } as const,
-            }
-          );
-
-          await step.sms(
-            'send-sms',
-            async (controls) => {
-              return {
-                body: `sms result ${payload.name}`,
-              };
-            },
-            {
-              controlSchema: {
-                type: 'object',
-                properties: {
-                  name: { type: 'string', default: 'TEST' },
-                },
-              } as const,
-            }
-          );
-        },
-        {
-          payloadSchema: {
-            type: 'object',
-            properties: {
-              name: { type: 'string', default: 'default_name' },
-            },
-            required: [],
-            additionalProperties: false,
-          } as const,
-        }
-      );
+    describe('100 times', () => {
```

Comment:
this needed to test the stability of the e2e?

Reply:
This was part of a test. I removed it.